deploy/helm/pulumi-operator/values.yaml

```diff
@@ -71,41 +63,39 @@ podAnnotations: {}
 # -- Deployment annotations
 deploymentAnnotations: {}
 
+# -- Service annotations
+serviceAnnotations: {}
+
 # -- Pod Security Context see [values.yaml](values.yaml)
 podSecurityContext:
-  # -- pulumi-kubernetes-operator group is 1000
-  fsGroup: 1000
-  # -- pulumi-kubernetes-operator user is 1000
-  runAsUser: 1000
+  runAsNonRoot: true
+  # -- pulumi-kubernetes-operator user is 65532
+  runAsUser: 65532
+  # -- pulumi-kubernetes-operator group is 65532
+  runAsGroup: 65532
+  # fsGroup: 65532
 
 # -- Security Context see [values.yaml](values.yaml)
 securityContext:
-  privileged: false
-  runAsNonRoot: true
   allowPrivilegeEscalation: false
   # readOnlyRootFilesystem: true
-  runAsGroup: 10003
-  # runAsUser: 10003
-  seccompProfile:
-    type: RuntimeDefault
+  # seccompProfile:
+  #   type: RuntimeDefault
   capabilities:
     drop:
       - ALL
 
-# -- containers which are run before the app containers are started
-initContainers: []
-
 # -- CPU/Memory resource requests/limits
 resources:
   # We usually recommend not to specify default resources and to leave this as a conscious
   # choice for the user. This also increases chances charts run on environments with little
   # resources, such as Minikube. If you do want to specify resources, uncomment the following
   # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
   limits:
-    cpu: 500m
-    memory: 5123Mi
+    cpu: 200m
+    memory: 128Mi
   requests:
-    cpu: 100m
+    cpu: 200m
```

Comment:
Rationale is to ensure that the pod is assigned the ["Guaranteed"](https://kubernetes.io/docs/concepts/workloads/pods/pod-qos/#guaranteed) QoS class.